wrappers/python/nntile/layer/linear.py

```diff
@@ -304,3 +304,39 @@ def from_torch(torch_linear, x, hidden_dim_tile, redux, next_tag):
             linear_nntile.b.value.from_array(torch_linear.bias.data.cpu().detach().numpy())
 
         return linear_nntile, next_tag
+
+    def get_layer_forward_flops(self):
+        x_shape = self.x.value.shape
+        w_shape = self.w.value.shape
+        if self.side == "L":
+            if self.trans_x:
+                return x_shape[0] * x_shape[1] * w_shape[1]
```

Comment:
parameter `self.ndim` is not taken into account. Our linear layer sometimes computes tensor dot with several axes.